### DISCUSSION TITLE
Apparent Bug: Document Title defaults to "Dashboard" for custom components

### DISCUSSION THREAD

**Discussion by wrobson-lllow** [payloadcms/payload#2926] [Feature Requests & Ideas]
I have several globals that have custom react components as the view. There seems to be a bug that doesn't pick up the "slug" of "label" to use as the document title when using a custom component. Instead of "collection name" appearing as the title, it defaults to "Dashboard". I attempted to set "document.title"  as the "slug" within the custom react component, however, upon load the correct title will appear before it is shortly overridden and replaced by "Dashboard."

![image](https://github.com/payloadcms/payload/assets/132296416/cfbb675c-db7f-4c49-89ad-452e3f2e86ac)

**Comment by JessRynkar** [comment]
Hey @wrobson-lllow - I responded over on the issue you opened [here](https://github.com/payloadcms/payload/issues/2961). If you need any further help, let us know!
